plugins/processors/noop/sample.conf

```diff
@@ -0,0 +1,24 @@
+# Do nothing processor
+[[processors.noop]]
+
+## Metric Filtering
+## The following options provide mechanisms to include or exclude entire
+## metrics. For specific details and examples see the metric filtering docs:
+## https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#metric-filtering
+
+## Metric Selectors - These will drop entire metrics
+## Filter on metric name or tag key + value
+# namepass = []
+# namedrop = []
+# tagpass = {}
+# tagdrop = {}
```

Comment:
Are you sure? As far as I can see, these will just ensure whether the metric will be feed to the processor. It will not actually drop the metric..

Reply:
If you disagree with something I really wish you would just put up a PR and not comment on closed and merged PRs

Reply:
I did not have time to look at it between creation and merging of this PR that closed a FR I have submitted.. Normally you aks the creator of the FR if the PR matches the expectations, no?

Anyway, did you try to drop a metric this way? I see the plugin is simply returning the provided metrics unconditionally, so I would be very amazed if this could suddenly drop the metric..